package.json

```
@@ -3,28 +3,20 @@
   "version": "0.0.1",
   "description": "A tree-sitter grammar for Elixir's \"IEx\" REPL syntax",
```

```suggestion
  "description": "Interactive Elixir (IEx) grammar for tree-sitter",
```

I like the new repo description text better than the old. What do you think?